geometry/src/main/java/coppercore/FieldFinder.java

```diff
@@ -0,0 +1,36 @@
+package coppercore.geometry;
+
+public class FieldFinder {
+    public Triangle triangle;
+
+    public FieldFinder(Triangle triangle) {
+        this.triangle = triangle;
+    }
+
+    public void setTriangle(Triangle triangle) {
+        this.triangle = triangle;
+    }
+
+    public boolean isPointInsideTriangle(Point point) {
+        Triangle t1 = new Triangle(point, triangle.getPoint2(), triangle.getPoint3());
+        Triangle t2 = new Triangle(triangle.getPoint1(), point, triangle.getPoint3());
+        Triangle t3 = new Triangle(triangle.getPoint1(), triangle.getPoint2(), point);
+
+        double area = triangle.getArea();
+
+        double sumOfT = t1.getArea() + t2.getArea() + t3.getArea();
+
+        return Math.abs(sumOfT - area) < 0.05;
+    }
```

Comment:
I feel like this function should be a part of a general `containsPoint` function that any `ConvexPolygon` (an abstract superclass name I just came up with) should be able to implement

Reply:
added